src/components/CallView/shared/Video.vue

```diff
@@ -275,6 +285,12 @@ export default {
 		},
 
 		participantName() {
+			if (this.model.attributes.subscribersOnlyPlaceholder) {
+				return t('spreed', 'Participants not allowed to publish: {number}', {
```

Comment:
debug code?

Reply:
No, the subscriber-only participants are hidden in the call view and replaced with a single participant with the count. I thought that it felt better for a "webcast" rather than having tens of passive participants in the call view.

Reply:
But thinking about our company call it will be a bit weird if there is only the speaker(s) and then a +X others and you can't even see quickly who is in the call so you can upgrade them to speak.

Reply:
As mentioned initially I thought that the company call would be a separate scenario in which everyone would be able to speak whenever they want, but the connections would be established only in that case. But if the company call is going to be handled with the `only moderators and those allowed by moderators can speak` approach then yes, I think that it would be easier if the call view was kept as is.

On the other hand, I think that hiding all the participants who can not publish (unless they have raised their hand) would be useful in more general presentation or speeches scenarios.

I will move the commits that change the call view to their own follow-up pull request to discuss it separately.